second session/assignment/shoppingcart.js

```diff
@@ -0,0 +1,134 @@
+document.addEventListener('DOMContentLoaded', function() {
+    var headerIcon = document.querySelector('.header-icon');
```

Comment:
p3) var의 여러가지 문제점 ( 재선언 가능, 함수레벨 스코프 ) 등 때문에, var 사용을 권장하지 않고 있어요! 
웬만하면 const를 사용하고, 값이 유동적으로 바뀌어야하는 상황에는 let을 사용하시는건 어떠세요~?
```suggestion
const headerIcon = document.querySelector('.header-icon');
```

Reply:
옙 var, let, const의 차이점 다시한번 복기해보겠습니다~

Reply:
p5: 저두 공부하고 갑니다~